src/select.html

```diff
@@ -13,4 +13,7 @@
 	<li ng-mousedown="updateModel(item)" ng-repeat="item in ::collection" ng-class="{'+selected': (selectedIndex === $index)}">
 		<a>{{appendText}} {{isString(item.value) ? item.value : item}}</a>
 	</li>
+	<li ng-if="linkText && linkHref">
+		<a href="{{linkHref}}" class="cps-blue-link">{{linkText}}</a>
+	</li>
```

Comment:
I ask cause there is some really annoying height stuff going on to position the dialog correctly, adding a link element might mess with the way the heights are calculated for absolute positioning: https://github.com/CanopyTax/cp-select/blob/master/src/select.js#L183